assets/playwright/index.html

```diff
@@ -7,6 +7,6 @@
   </head>
   <body>
     <div id="root"></div>
-    <script type="module" src="/playwright/index.{{extension}}"></script>
+    <script type="module" src="./index.{{extension}}"></script>
```

Comment:
Just to confirm, this is just a new default, the old value is still supported, correct?

Reply:
Correct, the old value is still supported.